terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/gh-runners.test.ts

```diff
@@ -669,7 +669,20 @@ runner_types:
       os: linux
       max_available: 1
       disk_size: 150
-      is_ephemeral: false`;
+      is_ephemeral: false
+    linux.4xlarge:
+      instance_type: c5.2xlarge
+      os: linux
+      max_available: 1
+      disk_size: 150
+      is_ephemeral: false
+      variants:
+        ephemeral:
+          is_ephemeral: true
+        large_disk:
+          disk_size: 300
+        amzn23:
+          ami: ami-123`;
```

Comment:
Cool idea. Is this expected to support variants with multiple properties (e.g. `large_disk` and `amzn2023` at the same time?

Reply:
Any variant, you can override any number of properties of the runner in the variant

Reply:
Not sure if I understood you correctly.  Given this variant definition, I'm asking if you want it to support invoking a label like "linux.2xlarge.ephemeral.amzn23"?

Note now amzn23 defines the ami while ephemeral adds it's own config

Reply:
No, you did not understand correctly, let me update the description.